sunpy/map/mapbase.py

```diff
@@ -1965,16 +1965,40 @@ def draw_rectangle(self, bottom_left, *, width: u.deg = None, height: u.deg = No
         axes.add_artist(rect)
         return [rect]
 
-    @u.quantity_input
-    def draw_contours(self, levels: u.percent, axes=None, **contour_args):
+    def _process_levels_arg(self, levels):
+        """
+        Accept a percentage or dimensionless or map unit input for contours.
```

Comment:
Would be good to add a `Returns` section (or equivalent sentence) to be clear that this returns the value to be contoured in `self.data`.